sycl/doc/design/OffloadDesign.md

```diff
@@ -208,9 +208,20 @@ The `sycl-post-link` tool is used after the device link is performed,
 applying any changes such as optimizations and code splitting before passing
 off to the `llvm-spirv` tool, which translates the LLVM-IR to SPIR-V.
 
-At this point, the SPIR-V binary is sent to the Ahead of Time compilation
-step to produce the final device binary or is wrapped before being linked into
-the final executable.
+|            Option                    |   Expected Behavior   |
+|--------------------------------------|-----------------------|
+| `--sycl-post-link-options=<arg>`     | Options that will control sycl-post-link step |
+| `--llvm-spirv-options=<arg>`         | Options that will control llvm-spirv step |
+
+*Table: Options to pass sycl-post-link and llvm-spirv options to the clang-linker-wrapper*
+
+Options that will be used by clang-linker-wrapper when invoking `sycl-post-link`
+tool are provided by the driver via the `--sycl-post-link-options=<arg>` option.
+Options that will be used by clang-linker-wrapper when invoking `llvm-spirv`
+tool are provided by the driver via the `--llvm-spirv-options=<arg>` option.
+
+SPIR-V binary is then sent to the Ahead of Time compilation step to produce the
+final device binary or is wrapped before being linked into the final executable.
```

Comment:
This isn't clear - we always wrap before being linked into the final executable.  The AOT step is the optional one based on the expected target.

Reply:
We actually do not need to mention this. Removing the sentence.

Thanks